### PR TITLE
createdRecipes Implementation with Firebase

### PR DESCRIPTION
**Pull request by jchen22002**
- Add Recipe Button
- Form popup
- Form include inputs such as title, description, ingredients and steps
- Need to implement image upload and display later

### CODE REVIEW

**Comment by Chr1sZzz** [comment]
Consider using any error handling to manage any unexpected situation. Also, might be creating a new repo to avoid the deployment stuff. Also, implement a feature that allows users to upload an image for their recipe, and then display that image alongside the recipe details, if possible.

**Comment by Chr1sZzz** [comment]
Missing the isopenform for the add-recipe. Maybe fix this later

File: src/app/userProfile/page.js [hunk 1]
```diff
@@ -4,53 +4,139 @@ import * as React from 'react';
 import styles from './userProfile.module.css';
 import ProfileImage from '@/components/ProfileImage/ProfileImage';
 import ProfileInfo from '@/components/ProfileInfo/ProfileInfo';
-//import Tabs from '@/components/Tabs/Tabs';
```
Comment (FreshPineapple-jpg) [inline src/app/userProfile/page.js:7]: Commented out code

**Review by FreshPineapple-jpg — APPROVED** [review]
One change you could add is to categorize some of the CSS elements into some classes.
The functionality seems good. Nice Work!

**Review by azuhayer — APPROVED** [review]
The functionality of your implementations look good! Safe to merge.

**Pull request merged**